app/src/main/java/chat/rocket/android/chatroom/ui/ChatRoomFragment.kt

```diff
@@ -792,6 +823,30 @@ class ChatRoomFragment : Fragment(), ChatRoomView, EmojiKeyboardListener, EmojiR
             button_add_reaction.setOnClickListener { view ->
                 openEmojiKeyboardPopup()
             }
+
+            button_drawing.setOnClickListener {
+                if (!canWriteToExternalStorage()) {
+                    checkWritingPermission()
+                }else{
+                    val intent = Intent(activity, DrawingActivity::class.java)
+                    startActivityForResult(intent, REQUEST_CODE_FOR_DRAW)
+                }
+                handler.postDelayed({
+                    hideAttachmentOptions()
+                }, 400)
+            }
+        }
+    }
+
+    private fun canWriteToExternalStorage(): Boolean {
```

Comment:
This is duplicate. We have `checkWritingPermission` function on the ImageHelper.kt

Please, remove it.